playground/frontend/playground_components_dev/lib/src/widget_tester.dart

```diff
@@ -50,4 +52,28 @@ extension WidgetTesterExtension on WidgetTester {
     final context = element(find.codeField());
     return context.read<PlaygroundController>();
   }
+
+  Future<void> navigateAndSettle(Uri url) async {
+    print(url);
```

Comment:
This unnamed print will be hard to find.

Reply:
Edited.